docs/source/models/supported_models.rst

```diff
@@ -177,51 +177,61 @@ Decoder-only Language Models
 
 .. _supported_vlms:
 
-Vision Language Models
-^^^^^^^^^^^^^^^^^^^^^^^
+Multimodal Language Models
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. list-table::
-  :widths: 25 25 50 5
+  :widths: 25 25 25 25 5
   :header-rows: 1
 
   * - Architecture
     - Models
+    - Supported Modality(ies)
```

Comment:
```suggestion
    - Supported Modalities
```
Imo this is easier to read.

Reply:
Ty for suggestion! I think I'll just keep it as is to avoid another doc build